website/content/en/docs/usage/resources/capapplicationversion.md

```diff
@@ -364,4 +366,4 @@ spec:
 >
 > The supported configurations is kept minimal intentionally to keep the overall API simple by considering commonly used configurations.
 
-> Note: For `initContainers` nearly the same environment variables as the main container are made available including VCAP_SERVICES environment.
\ No newline at end of file
+> Note: For `initContainers` nearly the same environment variables as the main container are made available including VCAP_SERVICES environment.
```

Comment:
could we also mention here that volume mounts (for service credentials, when env is set) would be made available on initContainers

Reply:
Updated in https://github.com/SAP/cap-operator/pull/72/commits/6a3e6a89927827d4d353b1fa797218abe6099d32